registry/server/common/services/assets/AssetsDiscoveryProcessor.ts

```diff
@@ -0,0 +1,44 @@
+import {
+    AssetsManifestReader,
+    AssetsManifest
+} from './AssetsManifestReader';
+
+export class AssetsDiscoveryProcessor {
```

Comment:
`Processor` looks not necessary, just `AssetsDiscovery` should be fine.

Reply:
I 100% agree. But we have already `AssetsDiscovery` https://github.com/namecheap/ilc/blob/master/registry/server/common/services/AssetsDiscovery.ts.

So I have added Processor suffix because it's responsibility to form URL from files from assets discovery